src/renderer/components/watch-video-info/watch-video-info.css

```diff
@@ -104,29 +119,23 @@
   .seperator {
     display: none;
   }
-
-  .videoButtons {
-    flex-direction: column;
-    align-items: flex-start;
-    margin-block-start: 10px;
-  }
-
-  .videoOptions {
-    flex-wrap: nowrap;
-  }
-
-  .videoOptions :deep(.iconDropdown) {
-    inset-inline: auto;
-  }
 }
 
 @media screen and (width <= 680px) {
   .likeSection {
     margin-inline-start: 0;
     text-align: start;
   }
+}
+
+@media screen and (width <= 730px) {
+  .videoButtons {
+    flex-direction: column;
+    align-items: flex-start;
+    margin-block-start: 10px;
+  }
 
-  .videoOptions :deep(.iconDropdown) {
+  .videoOptionsMobileRow :deep(.iconDropdown) {
     inset-inline: auto calc(50% - 20px);
```

Comment:
This style causes overflow on small width displays. It should probably float right in that case.
![2025-03-11-120924_hyprshot](https://github.com/user-attachments/assets/e10bc07e-b85a-4b35-92cd-dd0ec180ac8d)

IMHO, it would look better centered on large displays:
|how it looks|centered|
|--|--|
|![2025-03-11-121200_hyprshot](https://github.com/user-attachments/assets/4a415413-33f1-4b4a-bab2-b3634ba82c33)|![2025-03-11-121258_hyprshot](https://github.com/user-attachments/assets/277cc9bf-36a3-48ab-81ee-69e5414a7a1a)|

Additionally, this is nitpicky, but this should probably be handled through the existing props for `ft-icon-button` instead of deep styles.

Reply:
> this should probably be handled through the existing props

Resize handler to change `dropdownPositionX` prop?

Reply:
You know what. I didn't think about the resize handler in the JS side, and it is easier to handle responsiveness in CSS.